lib/assertions.js

```diff
@@ -390,7 +386,7 @@ module.exports = function (expect) {
                             // No common prefix, omit diff
                             return null;
                         } else {
-                            output.partialMatch(subject.substr(0, i)).text(subject.substr(i));
+                            output.match(subject.substr(0, i)).text(subject.substr(i), true);
```

Comment:
Yes I think so, it is much easier to understand a method name than a boolean parameter. Would that be okay with you.